packages/algob/src/builtin-tasks/test.ts

```diff
@@ -1,33 +1,25 @@
-import glob from "glob";
-import path from "path";
+import Mocha from "mocha";
 
-import { internalTask } from "../internal/core/config/config-env";
-import { cmpStr } from "../lib/comparators";
-import type { AlgobRuntimeEnv, ResolvedAlgobConfig } from "../types";
-import { TaskArgs } from "./deploy";
-import { TASK_TEST_GET_TEST_FILES } from "./task-names";
+import { task } from "../internal/core/config/config-env";
+import { testsDirectory } from "../lib/script-checkpoints";
+import type { AlgobConfig } from "../types";
+import { loadFilenames } from "./deploy";
+import { TASK_TEST } from "./task-names";
 
-export default function (): void {
-  internalTask(TASK_TEST_GET_TEST_FILES)
-    .addOptionalVariadicPositionalParam(
-      "filenames",
-      "An optional list of files to test",
-      []
-    )
-    .setAction(runTests);
-}
-
-function loadFiles (fileNames: string[], config: ResolvedAlgobConfig): string[] {
-  if (fileNames.length !== 0) {
-    return fileNames;
-  }
-  if (config.paths == null) {
-    throw new Error("unexpected non-project execution");
-  }
-  return glob.sync(path.join(config.paths.tests, "**/*.js")).sort(cmpStr);
+async function runTests (config: AlgobConfig): Promise<void> {
+  const testFiles = loadFilenames(testsDirectory);
+  console.log("Test files:", testFiles);
+  const mocha = new Mocha(config.mocha);
+  console.log("Adding test files to mocha");
+  testFiles.forEach((file) => mocha.addFile(file));
+  console.log("Executing test files ");
```

Comment:
Please remove these `console.logs`. If you think it's necessary to keep them, then we can use `console.info`/ `debug logs`. But i don't think we need them (You can just add comments to code, if not clear).

Reply:
I thought the command would be more informative by these statements, informing the user about every stage of the process

Reply:
`algob help` should be more than intuitive enough. Anyways this should be changed to `console.info` then (not `console.log`)